hack/e2e/metrics/override.yaml

```diff
@@ -0,0 +1,14 @@
+# This file contains override configurations for the cluster loader test.
+# It specifies various settings for provisioning volumes, waiting for PVs to be bound or deleted,
+# pod and volume configurations, and the expected provisioner and storage class.
+
+PROVISION_VOLUME: true
```

Comment:
nit: do we not have these defaulting to true? in [hack/e2e/metrics/cl2-config.yaml](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1624/files#diff-bcab731453d2b4b230a0181cad15f0dc0b6474f48e47f8e6b5dc7162772714e0)